src/main/java/com/snowflake/kafka/connector/SnowflakeSinkTask.java

```diff
@@ -172,18 +172,6 @@ public void start(final Map<String, String> parsedConfig) {
     final long bufferFlushTime =
         Long.parseLong(parsedConfig.get(SnowflakeSinkConnectorConfig.BUFFER_FLUSH_TIME_SEC));
 
-    // Falling back to default behavior which is to ingest an empty json string if we get null
-    // value. (Tombstone record)
-    SnowflakeSinkConnectorConfig.BehaviorOnNullValues behavior =
-        SnowflakeSinkConnectorConfig.BehaviorOnNullValues.DEFAULT;
-    if (parsedConfig.containsKey(SnowflakeSinkConnectorConfig.BEHAVIOR_ON_NULL_VALUES_CONFIG)) {
```

Comment:
BehaviorOnNullValues (aka ingest or don't ingest tombstone records) is only used in RecordService. Thus we don't need `setBehaviorOnNullValuesConfig` and can just pass the config into RecordService